internal/service/ldapverify/resource_ldap_verify_test.go

```diff
@@ -148,82 +142,49 @@ func importStateIDFunc(resourceName string) resource.ImportStateIdFunc {
 	}
 }
 
-func configBasic(projectName, orgID, clusterName, hostname, username, password string, port int) string {
+func configBasic(projectID, hostname, username, password string, port int) string {
 	return fmt.Sprintf(`
-		resource "mongodbatlas_project" "test" {
-			name   = %[1]q
-			org_id = %[2]q
-		}
-
-		resource "mongodbatlas_cluster" "test" {
-			project_id   = mongodbatlas_project.test.id
-			name         = %[3]q
-			
-			// Provider Settings "block"
-			provider_name               = "AWS"
-			provider_region_name        = "US_EAST_2"
-			provider_instance_size_name = "M10"
-			cloud_backup                = true //enable cloud provider snapshots
-		}
-
 		resource "mongodbatlas_ldap_verify" "test" {
-			project_id               =  mongodbatlas_project.test.id
-			hostname 				 = %[4]q
-			port                     =  %[7]d
-			bind_username            = %[5]q
-			bind_password            = %[6]q
-			depends_on = ["mongodbatlas_cluster.test"]
+			project_id               =  %[1]q
+			hostname 				 = %[2]q
+			port                     =  %[5]d
+			bind_username            = %[3]q
+			bind_password            = %[4]q
 		}
 	
 		data "mongodbatlas_ldap_verify" "test" {
 			project_id = mongodbatlas_ldap_verify.test.project_id
 			request_id = mongodbatlas_ldap_verify.test.request_id
 		}		
```

Comment:
while the ldap configuration does not use the created cluster in any way, if there is no cluster created the following error is encountered: `No clusters available to perform LDAP connectivity verification for groupId`